docs/source/topic/converting_pdf_to_text.rst

```diff
@@ -74,7 +74,7 @@ notion of the space character. A space is inserted if the characters are
 further apart that the `word_margin` (W in the figure). The `word_margin` is
 relative to the maximum width or height of the new character. Having a smaller
 `word_margin` creates smaller words. Note that the `word_margin` should at
-least be smaller than the `char_margin` otherwise none of the characters will
+least be bigger than the `char_margin` otherwise none of the characters will
```

Comment:
This might seem counter-intuitive but the word-margin needs to be smaller than the char-margin in order to be useful. 

In order to be separated by a space the characters need to be closer together than the char-margin, but further apart than the word-margin.

Reply:
So I guess `char_margin` is the maximum distance between chars for them to be chained together into a line. Any character further away than `char_margin` will be at another "line", like in another column in a multi-column layout.

Is that correct?